src/bitwise.rs

```diff
@@ -8,6 +8,8 @@ pub trait Bits {
     fn toggle_bit(&mut self, bit_idx: u8);
     fn set_bit(&mut self, bit_idx: u8, value: bool);
     fn get_bit(self, bit_idx: u8) -> bool;
+    fn clear_bits_msb_to_idx(&mut self, bit_idx: u8); // msb: most significant bit
+    fn clear_bits_idx_to_0(&mut self, bit_idx: u8);
```

Comment:
Where do you we need these right now?

Reply:
At the moment they will not be used but for the sake of completeness I have thrown them in.